ros/noetic.v3.17/ros-noetic-behaviortree-cpp-v3/APKBUILD

```diff
@@ -8,7 +8,7 @@ arch="all"
 license="MIT"
 
 depends="boost-dev ncurses-dev zeromq-dev boost-dev ncurses-dev zeromq-dev"
-makedepends="py3-setuptools py3-rosdep py3-rosinstall py3-rosinstall-generator py3-vcstool chrpath boost-dev ncurses-dev ros-noetic-ros-environment zeromq-dev"
+makedepends="py3-setuptools py3-rosdep py3-rosinstall py3-rosinstall-generator py3-vcstool chrpath boost-dev ncurses-dev ros-noetic-catkin>=0.5.78 ros-noetic-ros-environment ros-noetic-roslib zeromq-dev"
```

Comment:
Should by done by `apkbuild_hook.sh`
example: https://github.com/seqsense/aports-ros-experimental/blob/e642b17ae13617a34c0e4b31a2461ac4effa6965/ros/humble.v3.17/ros-humble-cyclonedds/apkbuild_hook.sh#L1